runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc

```diff
@@ -54,22 +55,31 @@ void ElementwiseActivationLayer::configure(const IPortableTensor *input, IPortab
     case ElementwiseActivationType::kReLU:
       if (input->data_type() == OperandType::FLOAT32)
       {
-        if (alpha == std::numeric_limits<float>::infinity() && beta == 0.f)
+        if ((alpha == std::numeric_limits<float>::infinity() || alpha == 6.0f) && beta == 0.f)
         {
           cpu::ops::ElementwiseActivationLayer::configure(
             input, output, alpha, beta, cpu::ops::ElementwiseActivationType::kReLU);
 
-          _backward_kernel = [](const IPortableTensor *output, const IPortableTensor *incoming,
-                                IPortableTensor *outgoing) {
-            nnfw::cker::train::ReLUGrad(getShape(output), getBuffer<float>(output),
-                                        getShape(incoming), getBuffer<float>(incoming),
-                                        getShape(outgoing), getBuffer<float>(outgoing));
+          auto relu_cker = [&alpha]() {
+            if (alpha == std::numeric_limits<float>::infinity())
+              return nnfw::cker::train::ReLUGrad;
+            else if (alpha == 6.0f)
```

Comment:
Q. Is it okay to compare float value with float value? I'm not sure....

Reply:
Could you explain a little more? 

At first glance, I failed to catch your worries.
I thought It is natural to compare one float with another.

Reply:
Generally not, but some range of integer is exactly represented in float format.

For float, it has 24-bit significand. 6 is exactly representalbe.

To make sure,

https://en.wikipedia.org/wiki/Single-precision_floating-point_format

> Integers between 0 and 16777216 can be exactly represented

Reply:
Almost it returns true

Reply:
Aha, I understood @YongseopKim's point.  
There is a floating-point (representataion) error issue.

Reply:
https://codereview.stackexchange.com/questions/275266/c-compare-floats

Reply:
FYI, this is just comment. Do not count me as the number of review.

Reply:
> Almost it returns true but rarely could return false. If you already know this and then code it, it would be okay. But if not, we could consider it. 😀

I don't understand. For the case in range (0 and 16777216), it is always safe to compare as I understand. Maybe I don't understand your point yet.

Reply:
As far as I know, `(6.0f == 6.0f)` is always true. They have the same precision since they are `float` literal.
But when one of them is a double or a long double like `(6.0f == 6.0)`, I'm not sure if they have the same precision.

Reply:
> It seems that both exponent value and fraction value of 6.0 and 6.0f are the same.

This is my point.

Reply:
>> It seems that both exponent value and fraction value of 6.0 and 6.0f are the same.
> This is my point.

I updated about it in `https://github.com/Samsung/ONE/pull/12487#discussion_r1456812278`. In short, if comparing value is 6.0, there is no problem.

Reply:
And I already told the term `approximate equality`.

if we have a consensus, because of value 6, it is okay. I'm curious what sth happens if the floating point's precision is too big.
- https://beta.boost.org/doc/libs/1_68_0/libs/math/doc/html/math_toolkit/float_comparison.html

Reply:
My comment is not pointing out that this is a problem.